test/src/test_argument_parser_info.cpp

```diff
@@ -1,6 +1,6 @@
 #define AP_TESTING
 
-#include "../doctest.h"
+#include "doctest.h"
 
 #include <ap/argument_parser.hpp>
 #include <argument_parser_test_fixture.hpp>
```

Comment:
Move the fixture include below the doctest include and use `""`.
This should look like this:
```
#include "doctest.h"
#include "argument_parser_test_fixture.hpp"

#include <ap/argument_parser.hpp>
```
To the same for all other test files

Reply:
Done.